README.md

```diff
@@ -89,112 +89,27 @@ directory (such as Jupyter Lab notebooks) will persist.
-aws sagemaker create-image-version \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --base-image $ECR_IMAGE_URI
-
-# Optional, step 2.1: Verify that the Image and Image Version were successfully created.
-# You may have to specify a different `--version-number` if you created multiple Image Versions.
-aws sagemaker describe-image-version \
-    --image-name $SAGEMAKER_IMAGE_NAME \
-    --version-number 1
-
-# Step 3.1: Create a file that will contain App Image Config parameters.
-cat > /tmp/app-config.json << EOF
-{
-   "AppImageConfigName": "app-image-config-$SAGEMAKER_IMAGE_NAME",
-   "KernelGatewayImageConfig": { 
-      "FileSystemConfig": { 
-         "DefaultGid": 100,
-         "DefaultUid": 1000,
-         "MountPath": "/home/sagemaker-user"
-      },
-      "KernelSpecs": [ 
-         { 
-            "DisplayName": "Python 3 (ipykernel)",
-            "Name": "python3"
-         }
-      ]
-   }
-}
-EOF
-
-# Step 3.2: Create an Amazon SageMaker App Image Config.
-aws sagemaker create-app-image-config \
-    --cli-input-json file:///tmp/app-config.json
-
-# Step 4.1: Create a file that will contain default parameters for User Settings.
-cat > /tmp/default-user-settings.json << EOF
-{
-    "DefaultUserSettings": {
-        "KernelGatewayAppSettings": {
-            "CustomImages": [
-                {
-                    "ImageName": "$SAGEMAKER_IMAGE_NAME",
-                    "AppImageConfigName": "app-image-config-$SAGEMAKER_IMAGE_NAME",
-                    "ImageVersionNumber": 1
-                }
-            ]
-        }
-    }
-}
-EOF
-
-# Step 4.2: Update Amazon SageMaker Domain with the new default User Settings.
-aws sagemaker update-domain \
-    --domain-id $SAGEMAKER_STUDIO_DOMAIN_ID \
-    --cli-input-json file:///tmp/default-user-settings.json
-```
-
-Now, for your User Profile, log-out and log back in to start using the custom image.
+To use the sagemaker-distribution image in SageMaker Studio, simply go to the image selection dropdown, and select `SageMaker Distribution v{Major_version} {CPU/GPU}`.
 
 ### "I want to directly use the Conda environment, not via a Docker image"
 
 Amazon SageMaker Distribution supports full reproducibility of Conda environments, so you don't necessarily need to use
 Docker. Just find the version number you want to use in the [build_artifacts](build_artifacts) directory, open one of
 _cpu.env.out_ or _gpu.env.out_ and follow the instructions in the first 2 lines.
 
+## Customizing image
+
+When installing packages onto the image, best practice would be to use `micromamba install --no-update-deps --channel conda-forge foo` rather than `pip install`, in order to prevent breaking the environment, while also avoiding unwanted changes to dependencies.
+
+If you would like to build a custom image using `sagemaker-distribution` as a base, this would look like:
+```
+FROM public.ecr.aws/sagemaker/sagemaker-distribution:latest-cpu
+
+sudo apt install ...
```

Comment:
isn't this apt-get ? 

And we should probably recommend this only for System level utility packages etc.